proof/crefine/X64/StoreWord_C.thy

```diff
@@ -50,16 +50,15 @@ lemma byte_to_word_heap_upd_outside_range:
    byte_to_word_heap (m (p := v')) base off = byte_to_word_heap m base off"
   apply (simp add: byte_to_word_heap_def Let_def)
   apply (erule contrapos_np)
-  apply (clarsimp intro!: intvl_inter_le [where k=0 and ka=7, simplified, OF refl]
-                          intvl_inter_le [where k=0 and ka=6, simplified, OF refl]
-                          intvl_inter_le [where k=0 and ka=5, simplified, OF refl]
-                          intvl_inter_le [where k=0 and ka=4, simplified, OF refl]
-                          intvl_inter_le [where k=0 and ka=3, simplified, OF refl]
-                          intvl_inter_le [where k=0 and ka=2, simplified, OF refl]
-                          intvl_inter_le [where k=0 and ka=1, simplified, OF refl]
-                          intvl_inter_le [where k=0 and ka=0, simplified, OF refl]
-                   split: if_split_asm)
-  done
+  by (clarsimp intro!: intvl_inter_le [where k=0 and ka=7, simplified, OF refl]
+                       intvl_inter_le [where k=0 and ka=6, simplified, OF refl]
+                       intvl_inter_le [where k=0 and ka=5, simplified, OF refl]
+                       intvl_inter_le [where k=0 and ka=4, simplified, OF refl]
+                       intvl_inter_le [where k=0 and ka=3, simplified, OF refl]
+                       intvl_inter_le [where k=0 and ka=2, simplified, OF refl]
+                       intvl_inter_le [where k=0 and ka=1, simplified, OF refl]
+                       intvl_inter_le [where k=0 and ka=0, simplified, OF refl]
+                split: if_split_asm) (* long, many if_asm splits *)
```

Comment:
does this change make much difference?

Reply:
Took me a while to dig this one out. Yes, absolutely worth it, this otherwise blocks for almost a full minute. I first thought something new was slowing this down, but it seems to always have been slow.